test-harness/README.md

```diff
@@ -24,6 +24,8 @@ Gold master files contain data about both request and response.
 `yarn test.binary`
 or
 `SPEC=./examples/petstore.oas2.json BINARY=./cli-binaries/prism-cli-linux yarn test.binary`
+or
```

Comment:
What happens when multiple specs are provided?

Reply:
https://github.com/stoplightio/prism/pull/307/files#diff-389656a19bda4636b1c0c52a8281822aR28

Reply:
Oh this is so cool! Sounds good.